test/cluster/test_tablets_merge.py

```
@@ -323,17 +329,19 @@ async def check():
 
 @pytest.mark.parametrize("racks", [2, 3])
 @pytest.mark.skip_mode(mode='release', reason='error injections are not supported in release mode')
-async def test_tablet_merge_cross_rack_migrations(manager: ManagerClient, racks):
+async def test_tablet_merge_cross_rack_migrations(manager: ManagerClient, racks,
+                                                  storage_config: FeatureConfig):
     cmdline = ['--target-tablet-size-in-bytes', '30000',]
-    config = {'tablet_load_stats_refresh_interval_in_seconds': 1}
+    config = storage_config.get_cluster_cfg({'tablet_load_stats_refresh_interval_in_seconds': 1})
     servers = []
     rf = racks
     for rack_id in range(0, racks):
```

Ruff (0.16.1)

[warning] Unnecessary `start` argument in `range`

Remove `start` argument

(PIE808)